docs/ng-install.md

```diff
@@ -0,0 +1,201 @@
+    angular2: path.resolve('node_modules/angular2'),
+    rxjs: path.resolve('node_modules/rxjs')
+  },
+  paths: {
+    '*': '*.js'
+  },
+  meta: {
+    'node_modules/angular2/*': { build: false },
+    'node_modules/rxjs/*': { build: false }
+  },
+};
+
+builder.config(config);
+
+builder
+.bundle(name, path.resolve(__dirname, 'bundles/', name + '.js'))
+.then(function() {
+  console.log('Build complete.');
+})
+.catch(function(err) {
+  console.log('Error', err);
+});
+````
+
+Just include this script in the root of your app named like `bundler-script.js` and run it before publishing your library to `npm`.
+
+Note that when TypeScript version 1.8 is out of `beta`, this script will no longer be needed and this documentation will be updated.
+
+#### Step 4
+
+Bump version, compile and republish.
+
+#### Step 5
+
+There is no step 5. It's that easy. Ok wait a second, what?!
+Let's break down the entry file mentioned above.
+
+```
+// We import these so we can reference them in the 'default' export below
+import {TestDirective} from './src/app/directives/test.directive';
+import {TestService} from './src/app/providers/test.provider';
+import {TestService2} from './src/app/providers/test2.provider';
+import {TestPipe} from './src/app/pipes/test.pipe';
+import {TestStyles} from './src/app/test.styles';
+
+// We export everything we want developers to be able to use manually
+export * from './src/app/directives/test.directive';
+export * from './src/app/providers/test.provider';
+export * from './src/app/providers/test2.provider';
+export * from './src/app/pipes/test.pipe';
+export * from './src/app/test.styles';
+
+// This is the magic.
+// Provides a standard way to export your library to allow angular-cli to help developers setup your library
+// Please note: keys are optional. Your library can provide any metadata is provides.
+export default {
+  directives: [TestDirective],
+  pipes: [TestPipe],
+  providers: [TestService, TestService2],
+  styles: TestStyles.styles(),
+  styleUrls: ['src/app/css/test.css']
+}
+```
+
+This would allow angular-cli to auto-annotate all relevant angular metadata that your library provides if the consumer chose to auto-inject.
+You can study this example library which provides this complete example:
+https://github.com/NathanWalker/ng2-cli-test-lib
+
+### Suggest a repo adhere to the Publisher Guide
+
+1. Create a new issue on the lib repo
+2. Copy/paste this subject: `Please add a default export to support angular-cli auto install feature`
+3. Copy/paste this message for the body:
+
+> It would be great if this library provided a `default` export object as described [here in the angular-cli Publisher's Guide](https://github.com/angular/angular-cli). Our project would greatly benefit from it and it's very easy to do. Please comment back if that would be possible. If you won't have time in the next couple days, I will try to submit a PR to provide this compatibility soon. Thank you!
```

Comment:
This should link to `https://github.com/angular/angular-cli/blob/master/docs/ng-install.md##b-publisher-guide-preparing-your-library`, I think.